### PR TITLE
Vet Clinic Database: Create animals table

### PR DESCRIPTION
**Pull request by amrendrakind**
###  Create animals table
- Use [this template](https://github.com/microverseinc/curriculum-template-databases) to generate your repo.
- Create a database named `vet_clinic`.
- Create a table named `animals` with the following columns:
    - ` id`: integer
    - `name`: string
    - `date_of_birth`: date
    - `escape_attempts`: integer
    - `neutered`: boolean
    - `weight_kg`: decimal
- Copy the SQL you used in the previous point into the schema.sql file in the generated repository.
- Insert the following data:
    - Animal: His name is Agumon. He was born on Feb 3rd, 2020, and currently weighs 10.23kg. He was neutered and he has never tried to escape.
    - Animal: Her name is Gabumon. She was born on Nov 15th, 2018, and currently weighs 8kg. She is neutered and she has tried to escape 2 times.
    - Animal: His name is Pikachu. He was born on Jan 7th, 2021, and currently weighs 15.04kg. He was not neutered and he has tried to escape once.
    - Animal: Her name is Devimon. She was born on May 12th, 2017, and currently weighs 11kg. She is neutered and she has tried to escape 5 times.
- Copy the SQL you used in the previous point into the data.sql file in the generated repository.
- Write queries for the following:
    - Find all animals whose name ends in "mon".
    - List the name of all animals born between 2016 and 2019.
    - List the name of all animals that are neutered and have less than 3 escape attempts.
    - List date of birth of all animals named either "Agumon" or "Pikachu".
    - List name and escape attempts of animals that weigh more than 10.5kg
    - Find all animals that are neutered.
    - Find all animals not named Gabumon.
    - Find all animals with a weight between 10.4kg and 17.3kg (including the animals with the weights that equals precisely 10.4kg or 17.3kg)
- Copy the SQL you used in the previous point into the queries.sql file in the generated repository.

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -1,3 +1,10 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM ANIMALS WHERE NAME LIKE '%mon';
+SELECT NAME FROM ANIMALS WHERE DATE_OF_BIRTH BETWEEN '2016-01-01' AND '2019-01-01';
```
Comment (KDlamini) [inline queries.sql]: Please make sure that the date is inclusive of 2019. Your current selection is only up to 1st January 2019, therefore excluding February to December of 2019. Kindly select date of birth up to 31st December 2019.

File: queries.sql [hunk 2]
```diff
@@ -1,3 +1,10 @@
...
+SELECT NAME FROM ANIMALS WHERE NEUTERED = '1' AND ESCAPE_ATTEMPTS < 3;
```
Comment (KDlamini) [inline queries.sql]: The value for NEUTERED can either be true or false (t or f in the database) therefore, the value 1 is invalid for neutered. Kindly use true instead.

File: queries.sql [hunk 3]
```diff
@@ -1,3 +1,10 @@
...
+SELECT DATE_OF_BIRTH FROM ANIMALS WHERE NAME = 'Agumon' OR NAME = 'Pikachu';
+SELECT NAME, ESCAPE_ATTEMPTS FROM ANIMALS WHERE WEIGHT_KG > 10.5;
+SELECT * FROM ANIMALS WHERE NEUTERED = '1';
```
Comment (KDlamini) [inline queries.sql]: Same here, kindly use true instead of 1.

**Review by KDlamini — CHANGES_REQUESTED** [review]
### Hi @amrendrakind 🙋‍♀️,  

Great work so far! 💯 
There are some issues that you still need to work on improving your project according to the requirements. You are almost there! 💪🏾 

## Required Changes ♻️
 
**Kindly look through the comments below 👇 for required changes**

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

N/A

Cheers 🥂  and Happy coding!!! 👯 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @KDlamini  in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by gemmen29 — APPROVED** [review]
Hi @amrendrakind, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 💯 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 



## Optional suggestions

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**